plugins/modules/remote_management/manageiq/manageiq_policies.py

```diff
@@ -292,8 +296,9 @@ def main():
     actions = {'present': 'assign', 'absent': 'unassign', 'list': 'list'}
     argument_spec = dict(
         policy_profiles=dict(type='list'),
-        resource_name=dict(required=True, type='str'),
-        resource_type=dict(required=True, type='str',
+        resource_id=dict(required=False, type='int'),
+        resource_name=dict(required=False, type='str'),
+        resource_type=dict(required=False, type='str',
```

Comment:
Is change of resource_type to optional deliberate?  If yes, please update doc on L38.  
But I think it should remain mandatory — I don't think you can infer type from id, an id is only meaningful given type, and anyway the code change doesn't try to handle missing type.

Same comment about 2nd module.

Reply:
You're right. resource_type needs to be required